components/match2/wikis/splitgate/match_group_input_custom.lua

```diff
@@ -0,0 +1,556 @@
+---
+-- @Liquipedia
+-- wiki=splitgate
+-- page=Module:MatchGroup/Input/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+
+local MatchGroupInput = Lua.import('Module:MatchGroup/Input', {requireDevIfEnabled = true})
+
+local ALLOWED_STATUSES = { 'W', 'FF', 'DQ', 'L', 'D' }
+local MAX_NUM_OPPONENTS = 8
+local MAX_NUM_PLAYERS = 10
+local MAX_NUM_VODGAMES = 9
+local MAX_NUM_MAPS = 9
+local DEFAULT_BESTOF = 3
+local ERROR_NUM = -99
```

Comment:
`MIN_SCORE` maybe?

Reply:
or `_NO_SCORE` as it indicates no score being present